src/core/components/forms/radioGroup/radioGroup/radioGroup.tsx

```diff
@@ -1,8 +1,29 @@
-import { RadioGroup as PrimitiveRadioGroup, type RadioGroupProps } from '@radix-ui/react-radio-group';
+import { RadioGroup as PrimitiveRadioGroup } from '@radix-ui/react-radio-group';
 import classNames from 'classnames';
-import { forwardRef } from 'react';
+import { forwardRef, type ComponentProps } from 'react';
 
-export interface IRadioGroupProps extends Omit<RadioGroupProps, 'orientation'> {}
+export interface IRadioGroupProps extends Omit<ComponentProps<'div'>, 'orientation' | 'dir'> {
```

Comment:
We don't probably need to omit `orientation` prop now
```suggestion
export interface IRadioGroupProps extends Omit<ComponentProps<'div'>, 'dir'> {
```

Reply:
✅ https://github.com/aragon/ods/pull/218/commits/c5cb9f813f7f531f183e261df873c6d87e4efd31